src/internal/constants.ts

```diff
@@ -8,6 +8,10 @@ export const CLASS_DRAG_IMAGE = CLASS_PREFIX + "drag-image";
 export const CLASS_DRAG_IMAGE_SNAPBACK = CLASS_PREFIX + "snapback";
 export const CLASS_DRAG_OPERATION_ICON = CLASS_PREFIX + "icon";
 
+// custom event
+export const EVENT_PREFIX = "dnd-poly-event-";
```

Comment:
Just a nitpick, but could we change the prefix to `dnd-poly-` because I think it's redundant to have the word `event` in an event.